api/turing/imagebuilder/imagebuilder.go

```diff
@@ -384,6 +370,58 @@ func (ib *imageBuilder) createKanikoJob(
 	)
 }
 
+func (ib *imageBuilder) configureKanikoArgsToAddCredentials(kanikoArgs []string) []string {
+	if ib.imageBuildingConfig.KanikoConfig.PushRegistryType == "gcr" || ib.artifactServiceType == "gcs" {
```

Comment:
sorry i missed this previously, but can you add the `gcr` and `gcs` in the constant variable?

Reply:
Ah good point, done! :D